main.ts

```diff
@@ -46,5 +48,12 @@ export default async (cwd: Dir, opts: Options) => {
 
   const output = results.map(r => criteria(r));
 
-  writer(output);
+  writer(output, {
+    errorsOnly: opts.usingCI,
+    isVerbose: opts.isVerbose
+  });
+
+  if (opts.usingCI && output.some(reportHasError)) {
```

Comment:
By default eslint exits with non-zero, maybe we should stick to that behavior? Thoughts?

Reply:
I guess I had one eye on integrating this early without erroring on "errors" but perhaps that makes more sense :+1: